samples/java/com/microsoft/bing/commerce/samples/IngestionSampleCode.java

```diff
@@ -0,0 +1,248 @@
+package com.microsoft.bing.commerce.samples;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.ArrayList;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.sym.Name;
+
+import okhttp3.OkHttpClient;
+import retrofit2.Retrofit;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import com.microsoft.bing.commerce.ingestion.BingCommerceIngestion;
+import com.microsoft.bing.commerce.ingestion.implementation.BingCommerceIngestionImpl;
+import com.microsoft.bing.commerce.ingestion.models.*;
+
+import com.microsoft.bing.commerce.search.BingCommerceSearch;
+import com.microsoft.bing.commerce.search.implementation.BingCommerceSearchImpl;
+import com.microsoft.bing.commerce.search.models.*;
+import com.microsoft.bing.commerce.search.util.AccessTokenInterceptor;
+import com.microsoft.bing.commerce.search.util.AccessTokenProvider;
```

Comment:
Why need the search imports ? In general please review all imports to make sure to only keep the necessary ones...